ckan/lib/mailer.py

```diff
@@ -70,7 +67,8 @@ def _mail_recipient(
     msg['From'] = _("%s <%s>") % (sender_name, mail_from)
     msg['To'] = u"%s <%s>" % (recipient_name, recipient_email)
     msg['Date'] = utils.formatdate(time())
-    msg['X-Mailer'] = "CKAN %s" % ckan.__version__
+    if not asbool(config.get('ckan.hide_version')):
```

Comment:
Since https://github.com/ckan/ckan/pull/6467, if you have declared a config option it will automatically get validated and parsed if you use `config.get_value()`
```suggestion
    if not config.get('ckan.hide_version'):
```

Reply:
That's an interesting feature, but your suggested change doesn't use `get_value`?